internal/lock.go

```diff
@@ -150,28 +242,32 @@ func (l *Lock) Download(dir string, tags []string, notags []string, perm string,
 		return fmt.Errorf("nothing to download")
 	}
 	errorCh := make(chan error, total)
-
-	var statusLine *StatusLine
-	if status {
-		statusLine = NewStatusLine(ctx, &filteredResources)
-		err := statusLine.InitResourcesSizes()
-		if err == nil {
-			statusLine.Start(true)
-		} else {
-			statusLine = nil // Do not update or display SL.
-		}
-	}
-
-	for i, r := range filteredResources {
```

Comment:
Was this code just added in #24?  If so, you shouldn't be removing it.

Reply:
Yes, this looks like the result of a bad conflict resolution. You shouldn't be removing this code.